packer/install-dependencies.sh

```diff
@@ -92,7 +92,7 @@ for TILLER_VERSION in ${TILLER_VERSIONS}; do
     pullContainerImage "docker" "gcr.io/kubernetes-helm/tiller:v${TILLER_VERSION}"
 done
 
-CLUSTER_AUTOSCALER_VERSIONS="1.13.1 1.12.1 1.3.4 1.3.3 1.2.2 1.1.2"
+CLUSTER_AUTOSCALER_VERSIONS="1.13.1 1.12.2 1.3.4 1.3.3 1.2.2 1.1.2"
```

Comment:
is 1.12.1 not used anywhere anymore?

Reply:
cluster-autoscaler 1.12.1 was only used by k8s 1.12.x, so as far as I can tell this patch release replaces it.